packages/ui/fe-bundle/src/utils/theme/loadDocsTypography.ts

```diff
@@ -0,0 +1,122 @@
+import type * as FernRegistryDocsRead from "@fern-fern/registry-browser/api/resources/docs/resources/v1/resources/read";
+
+export function getFontExtension(url: URL): string {
```

Comment:
nbd but generally I default to not exporting things unless i need them in other files, makes it easier to change impl detail without breaking consumers